Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -6688,6 +6754,12 @@
             "state" : "translated",
             "value" : "NZ / Australia"
           }
+        },
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "뉴질랜드/호주"
```

Comment:
@mplorentz Contrary to the above, this one _should_ be translated into the user's native language, I think, since it's for accounts from this region. Do you agree?

Reply:
yeah